data-esnext.js

```diff
@@ -1902,7 +1902,7 @@ exports.tests = [
       name: 'private fields initializers',
       exec: function () {/*
         class C {
-          #x = 42;
+          this.#x = 42;
```

Comment:
oh crap sorry, this is my bad :-( the shorthand is `#x` in code - but in the class body, `#x = 42` is already correct (－‸ლ) apologies